### PR TITLE
add dependency file

### PR DESCRIPTION
**Pull request by Liaojinghui**
Add dependency file for `neo-cli` to install plugin dependencies automatically.

### CODE REVIEW

File: src/ApplicationLogs/ApplicationLogs/DEPENDENCY [hunk 1]
```diff
@@ -0,0 +1 @@
+RpcServer
```
Comment (erikzhang) [inline src/ApplicationLogs/ApplicationLogs/DEPENDENCY:1]: Move it to `config.json`? I don't think it is worth using a separate file.

File: src/ApplicationLogs/LogReader.cs [hunk 2]
```diff
@@ -30,7 +30,14 @@ protected override void Configure()
...
         {
             if (system.Settings.Network != Settings.Default.Network) return;
-            RpcServerPlugin.RegisterMethods(this, Settings.Default.Network);
+            try
+            {
+                RpcServerPlugin.RegisterMethods(this, Settings.Default.Network);
+            }
+            catch
+            {
+                Console.WriteLine("Please run `install RpcServer` to install RpcServer Plugin.");
```
Comment (shargon) [inline src/ApplicationLogs/LogReader.cs:39]: Just please install? Maybe the command change in a future

Comment (Liaojinghui) [reply src/ApplicationLogs/LogReader.cs:39]: Just in case, it is not expected to be triggered.

Comment (erikzhang) [reply src/ApplicationLogs/LogReader.cs:39]: I think we don't need this `try`.

Comment (Liaojinghui) [reply src/ApplicationLogs/LogReader.cs:39]: > I think we don't need this `try`.

You can never make sure that the dependency always exists, if they don't, `file not exists exception` will throw and the software will crash directly without telling users how to fix it.

**Review by shargon — APPROVED** [review]

**Comment by Liaojinghui** [comment]
@superboyiii

**Comment by cloud8little** [comment]
Install RpcServer and ApplicationLogs, and then delete RpcServer.dll, and start neo-cli. Seems I can not find this message "Please run `install RpcServer` to install RpcServer Plugin.". 

2f731eca94fe9324ea3bf828cbf65b8

**Comment by Liaojinghui** [comment]
> Install RpcServer and ApplicationLogs, and then delete RpcServer.dll, and start neo-cli. Seems I can not find this message "Please run `install RpcServer` to install RpcServer Plugin.". @Liaojinghui
> 

install command not working for this or yet, its not included in the released plugin.

**Comment by cloud8little** [comment]
> > Install RpcServer and ApplicationLogs, and then delete RpcServer.dll, and start neo-cli. Seems I can not find this message "Please run `install RpcServer` to install RpcServer Plugin.". @Liaojinghui
> 
> @cloud8little install command not working for this pr yet, its not included in the released plugin.

I build the ApplicationLogs.dll using this PR.

**Review by erikzhang — APPROVED** [review]

**Comment by cloud8little** [comment]
Test passed, build ApplicationLogs/OracleService/StateService/RpcNep17Tracker will install RpcServer.

**Review by cloud8little — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**